services/web/server/src/simcore_service_webserver/security/_authz_db.py

```diff
@@ -35,8 +35,12 @@ async def get_active_user_or_none(engine: Engine, email: str) -> AuthInfoDict |
             )
         )
         row = await result.fetchone()
-        assert row is None or parse_obj_as(IdInt, row.id) is not None  # nosec
-        assert row is None or parse_obj_as(UserRole, row.role) is not None  # nosec
+        assert (
+            row is None or TypeAdapter(IdInt).validate_python(row.id) is not None
+        )  # nosec
```

Comment:
NOTE: these `# nosec` must be **on the same line** as `assert`. 
For the moment, they do not have an effect but if we re-activate security checks it will fail.
Let's try to keep the correctly ... otherwise we will have to review them again